README.md

```diff
@@ -108,13 +108,13 @@ Replace the contents of `villains/villain.service.ts` with the following code.
 
 ```typescript
 import { Injectable } from '@angular/core';
-import { EntityServiceBase, EntityServiceFactory } from 'ngrx-data';
+import { EntityCollectionServiceBase, EntityCollectionServiceFactory } from 'ngrx-data';
 import { Villain } from '../core';
 
 @Injectable()
-export class VillainService extends EntityServiceBase<Villain> {
-  constructor(entityServiceFactory: EntityServiceFactory) {
-    super('Villain', entityServiceFactory);
+export class VillainService extends EntityCollectionServiceBase<Villain> {
+  constructor(entityCollectionServiceFactory: EntityCollectionServiceFactory) {
```

Comment:
This should be: `EntityCollectionServiceElementsFactory` as it's used in the [villains service](https://github.com/johnpapa/angular-ngrx-data/blob/df9d4279ebc7f079fc56ed18e29ca0aa6836487a/src/app/villains/villains.service.ts#L31) in angular-ngrx-data: